edx_solutions_organizations/models.py

```diff
@@ -28,6 +28,9 @@ class Organization(TimeStampedModel):
     # could be different for each organization
     attributes = models.TextField(default='{}')
     include_manager_info = models.BooleanField(default=False)
+    is_redirected_to_docebo = models.BooleanField(default=False)
+    is_sso_enabled = models.BooleanField(default=False)
+    redirect_url = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
Lets make field names descriptive: 
`migrated_to_docebo`
`docebo_sso_enabled`
`docebo_redirect_url`

`docebo_redirect_url` could be a `models.URLField`